pkgs/development/compilers/elm/packages/elm-test-rs.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, curl, rustPlatform, fetchurl, openssl, stdenv, Security, darwin }:
+rustPlatform.buildRustPackage rec {
+  pname = "elm-test-rs";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://github.com/mpizenberg/elm-test-rs/archive/v${version}.tar.gz";
+    sha256 = "sha256:1manr42w613r9vyji7pxx5gb08jcgkdxv29qqylrqlwxa8d5dcid";
+  };
+
+  buildInputs = [ curl openssl ] ++ lib.optionals stdenv.isDarwin [ Security darwin.apple_sdk.frameworks.CoreServices ];
```

Comment:
I believe even curl dependency in here is not required. We should keep scope as small as possible so unless there is a reason to keep it I would remove it.

Reply:
I had tried removing it, but it did not work. Maybe I have done it wrong? Were you successful removing it from the deps?

Reply:
It is the first time I contribute to Nixpkgs, and I do not know very well what it is the best way to build locally. Maybe you have good pointers for that?

Reply:
from my shell:

```
marek@nixos-mainframe λ ~/Projects/nixpkgs (nixpkgs-review/1●)   
❯❯❯ <pkgs/development/compilers/elm/packages/elm-test-rs.nix | grep curl

marek@nixos-mainframe λ ~/Projects/nixpkgs (nixpkgs-review/1●)   
❯❯❯ nix-build -A elmPackages.elm-test-rs
error: anonymous function at /home/marek/Projects/nixpkgs/pkgs/development/compilers/elm/packages/elm-test-rs.nix:1:1 called with unexpected argument 'curl', at /home/marek/Projects/nixpkgs/pkgs/development/compilers/elm/default.nix:115:19

marek@nixos-mainframe λ ~/Projects/nixpkgs (nixpkgs-review/1●)   
❯❯❯ nix-build -A elmPackages.elm-test-rs
/nix/store/x4zi2khnwsix4l49d4hdn3hxzjn9z51q-elm-test-rs-2.0

marek@nixos-mainframe λ ~/Projects/nixpkgs (nixpkgs-review/1●)   
❯❯❯ ./result/bin/elm-test-rs --version
elm-test-rs 2.0.0
```

this is the patch:

```diff
diff --git a/pkgs/development/compilers/elm/default.nix b/pkgs/development/compilers/elm/default.nix
index 8b840f8945f..a4b0d12fd8b 100644
--- a/pkgs/development/compilers/elm/default.nix
+++ b/pkgs/development/compilers/elm/default.nix
@@ -113,7 +113,7 @@ let
     };
 
     elm-test-rs = import ./packages/elm-test-rs.nix {
-      inherit curl lib rustPlatform fetchurl openssl stdenv Security darwin;
+      inherit lib rustPlatform fetchurl openssl stdenv Security darwin;
     } // {
       meta = with lib; {
         description = "Fast and portable executable to run your Elm tests";
diff --git a/pkgs/development/compilers/elm/packages/elm-test-rs.nix b/pkgs/development/compilers/elm/packages/elm-test-rs.nix
index de06a1f4acd..a482809ab50 100644
--- a/pkgs/development/compilers/elm/packages/elm-test-rs.nix
+++ b/pkgs/development/compilers/elm/packages/elm-test-rs.nix
@@ -1,4 +1,4 @@
-{ lib, curl, rustPlatform, fetchurl, openssl, stdenv, Security, darwin }:
+{ lib, rustPlatform, fetchurl, openssl, stdenv, Security, darwin }:
 rustPlatform.buildRustPackage rec {
   pname = "elm-test-rs";
   version = "2.0";
@@ -8,7 +8,7 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256:1manr42w613r9vyji7pxx5gb08jcgkdxv29qqylrqlwxa8d5dcid";
   };
 
-  buildInputs = [ curl openssl ] ++ lib.optionals stdenv.isDarwin [ Security darwin.apple_sdk.frameworks.CoreServices ];
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security darwin.apple_sdk.frameworks.CoreServices ];
 
   cargoSha256 = "sha256:1dpdlzv96kpc25yf5jgsz9qldghyw35x382qpxhkadkn5dryzjvd";
   verifyCargoDeps = true;

```

Reply:
I just force-pushed the version without `curl`.